packages/lib/src/interactions/InteractionsProvider.tsx

```diff
@@ -43,12 +43,14 @@ function InteractionsProvider(props: { children: ReactNode }) {
     (interactionId: string, event: MouseEvent | WheelEvent) => {
       const registeredInteractions = [...interactionMap.values()];
 
-      function isButtonPressed(button: MouseButton | 'Wheel') {
+      function isButtonPressed(button: MouseButton | MouseButton[] | 'Wheel') {
```

Comment:
I'm not bothering to normalise `button` into an array like with `modifierKey`, since it is only used below.